parsing/parsetree.mli

```diff
@@ -211,6 +211,12 @@ and object_field_desc =
 and pattern =
     {
      ppat_desc: pattern_desc;
+     (** Prefer [Extensions.Pattern.get_desc] instead of using
+         this field directly, as the former will detect extension nodes
+         correctly. Extensions are encoded as
+         [Ppat_tuple [Ppat_extension _; _]]; if your pattern match avoids
+         matching that pattern, it is OK to skip [get_desc]. *)
+
```

Comment:
Do we want to mark this comment (and the analogous later one) as being `ocaml-jst` specific?